fastlane/Fastfile

```diff
@@ -159,14 +164,17 @@ def attach_changelog_to_main(options)
   current_changelog_data = current_changelog.read
   master_changelog_data = master_changelog.read
 
-  current_changelog.close  
+  current_changelog.close
   master_changelog.close
 
   File.open("../CHANGELOG.md", 'w') { |master_changelog_write_mode|
     version_header = "## #{version_number}"
     whole_file_data = "#{version_header}\n\n#{current_changelog_data}\n#{master_changelog_data}"
     puts "going to save. Contents - #{whole_file_data}"
-    
     master_changelog_write_mode.write(whole_file_data)
   }
 end
+
+def is_snapshot_version?(version_name)
+  version.end_with?("-SNAPSHOT")
```

Comment:
@beylmk this lane is making the lane crash since it should be `version_name` instead of `version` https://app.circleci.com/pipelines/github/RevenueCat/purchases-android/1764/workflows/f8624ce4-af0a-4971-93e2-43ef569876b5/jobs/4489